src/Microsoft.ML.TimeSeries/IidChangePointDetector.cs

```diff
@@ -139,9 +181,106 @@ public override void Save(ModelSaveContext ctx)
             base.Save(ctx);
         }
 
-        public IDataTransform ApplyToData(IHostEnvironment env, IDataView newSource)
+        // Factory method for SignatureLoadRowMapper.
+        private static IRowMapper Create(IHostEnvironment env, ModelLoadContext ctx, ISchema inputSchema)
+            => Create(env, ctx).MakeRowMapper(inputSchema);
+    }
+
+    public sealed class IidChangePointEstimator : IEstimator<ITransformer>
```

Comment:
>IEstimator [](start = 50, length = 10)

actually, since this is not trainable, it should probably be a `TrivialEstimator<IidChangePointDetector>`

Reply:
great suggestion, thx, done

---
In reply to: [226105519](https://github.com/dotnet/machinelearning/pull/1254#discussion_r226105519) [](ancestors = 226105519)